src/dotnet/commands/dotnet-new/FSharp_Console/project.json.template

```diff
@@ -9,31 +10,18 @@
       ]
     }
   },
-  "dependencies": {
-    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160509"
-  },
   "tools": {
-    "dotnet-compile-fsc": {
-      "version": "1.0.0-preview2-*",
-      "imports": [
-        "dnxcore50",
-        "portable-net45+win81",
-        "netstandard1.3"
-      ]
-    }
+    "dotnet-compile-fsc":"1.0.0-preview2-*"
   },
   "frameworks": {
     "netcoreapp1.0": {
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
           "version": "1.0.0"
-        }
-      },
-      "imports": [
-        "portable-net45+win8",
-        "dnxcore50"
-      ]
+        },
+        "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629"
```

Comment:
@catermp that can be resolved incrementing the `alpha` to `beta` (or `alpha2`). At least like that using `-alpha-*` is possibile to upgrade it, because dotnet-new template atm is fixed.
but np, i just fear more a fixed template, because a package can be reverted easy, the template is embedded.
but i understand fixed is safer, anyway `dotnet-new` is getting templates from url, maybe next version? :smiley: